frontend/packages/ceph-storage-plugin/src/components/create-storage-system/create-storage-system-steps/security-and-network-step/encryption.tsx

```diff
@@ -142,9 +142,24 @@ export const Encryption: React.FC<EncryptionProps> = ({ encryption, kms, dispatc
     encryption.clusterWide || encryption.storageClass,
   );
 
+  React.useEffect(() => {
+    if (isMCG) {
+      setEncryptionChecked(true);
+      dispatch({
+        type: 'securityAndNetwork/setEncryption',
+        payload: {
+          ...encryption,
+          clusterWide: true,
+          hasHandled: true,
+        },
+      });
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
   React.useEffect(() => {
     // To add validation message for encryption
-    if (!encryption.clusterWide && !encryption.storageClass && encryptionChecked) {
+    if (!encryption.clusterWide && !encryption.storageClass && encryptionChecked && !isMCG) {
```

Comment:
This hook updates validations which is not required for MCG.
Since with MCG only, storage class encryption not supported